examples/case-study-1/TODO

```diff
@@ -0,0 +1,7 @@
+Items or features to be done:
+
++ split models in 'load-db.p6' into packages in the 'lib' subdirectory
```

Comment:
Doing this will provide an additional canary test against new precompilation problems as well demonstrating that stuff can be organised sensibly.